k9mail/src/main/res/values/strings.xml

```diff
@@ -1203,6 +1203,7 @@ Please submit bug reports, contribute new features and ask questions at
     <string name="recipient_error_non_ascii">Special characters are currently not supported!</string>
     <string name="recipient_error_parse_failed">Error parsing address!</string>
     <string name="account_settings_crypto_support_sign_only">Support signing of unencrypted messages</string>
+    <string name="account_settings_crypto_support_default_opportunistic">Try to encrypt every message</string>
```

Comment:
I don't particularly like the word "try" here, it implies failure. We should also avoid "opportunistic" too, it's a needlessly complicated term. How about "Encrypt by default"? Or the other way around, "Disable automatic encryption"?

More opinions on this? @dschuermann @cketti @philipwhiuk